ethereum/statetransition/src/main/java/tech/pegasys/artemis/statetransition/util/EpochProcessorUtil.java

```diff
@@ -439,13 +439,11 @@ public static void attestionInclusion(BeaconState state, UnsignedLong previous_t
    */
   public static void crosslinkRewards(BeaconState state, UnsignedLong previous_total_balance)
       throws Exception {
-    UnsignedLong current_epoch = BeaconStateUtil.get_current_epoch(state);
     UnsignedLong previous_epoch = BeaconStateUtil.get_previous_epoch(state);
     List<Integer> slot_range =
         IntStream.range(0, Constants.EPOCH_LENGTH).boxed().collect(Collectors.toList());
-    List<PendingAttestation> attestations =
-        AttestationUtil.get_epoch_attestations(state, current_epoch);
-    List<Integer> attester_indices = AttestationUtil.get_attester_indices(state, attestations);
+    List<Integer> attester_indices =
+        AttestationUtil.get_previous_epoch_boundary_attester_indices(state);
```

Comment:
Why are we using previous_epoch_boundary_attester_indices here? I don't see attester_indices being mentioned in the spec for this function.

Reply:
https://github.com/ethereum/eth2.0-specs/blob/v0.1/specs/core/0_beacon-chain.md#crosslinks-1
`If index not in attesting_validators(crosslink_committee), state.validator_balances[index] -= base_reward(state, index).`